hera_mc/cm_handling.py

```diff
@@ -251,8 +251,13 @@ def show_parts(self, part_dossier, verbosity='h'):
                         ptsout += k + ', '
                     tdata.append(ptsin.strip().strip(','))
                     tdata.append(ptsout.strip().strip(','))
-                    if (part_dossier[hpnr]['part_info'] is not None):
-                        comment = part_dossier[hpnr]['part_info'].comment
+                    if len(part_dossier[hpnr]['part_info']):
+                        comment = ''
+                        for pi in part_dossier[hpnr]['part_info']:
+                            if not len(comment):
+                                comment = pi.comment
+                            else:
+                                comment = comment + ', ' + pi.comment
```

Comment:
This can all be simplified to `comment = ', '.join(part_dossier[hpnr]['part_info'])`.

Reply:
Actually, I only want the .comment field from it so I think I have to loop.